ports/stm32f4xx/dlmstp.c

```diff
@@ -175,6 +175,12 @@ static unsigned ReceivePDUCount;
 static volatile struct dlmstp_packet PDU_Buffer[MSTP_PDU_PACKET_COUNT];
 static RING_BUFFER PDU_Queue;
 
+/* Callback function to be called every time we receive a preamble */
+static dlmstp_hook_frame_rx_start_cb PreamCallback = NULL;
```

Comment:
Please rename `PreamCallback` to `Preamble_Callback`
Abbreviations may unintentionally encrypt the words and not be understandable by others.
Underscores between words improve readability. Yep, it's a double standards as the BACnet standard is full of words put together without underscores that are brought into this codebase as-is.

Reply:
Fixed this.  Also followed suit by renaming `FrameRxCallback` to `Frame_Rx_Callback`